diff --git a/lottery/src/schema.rs b/lottery/src/schema.rs
index 7b55d2a..e18d1e3 100644
--- a/lottery/src/schema.rs
+++ b/lottery/src/schema.rs
@@ -21,9 +21,9 @@ diesel::table! {
         title -> Text,
         #[sql_name = "type"]
         type_ -> Int4,
-        items -> Array<Nullable<Item>>,
+        items -> Array<Item>,
         remark -> Bool,
-        remarks -> Array<Nullable<Remark>>,
+        remarks -> Array<Remark>,
         created_at -> Timestamp,
         updated_at -> Timestamp,
     }
